api/admin.php

```diff
@@ -27,6 +27,11 @@
         unlink('../mail-addresses.txt');
     }
 
+    // delete personal config
+    if(is_file('../config/my.config.inc.php')){
+        unlink('../config/my.config.inc.php');

```

Comment:
If we delete the config file we should rename the reset button into factory reset or something, because I think you don't expect that also your config is deleted. The best solution would be to have a list of things that are deleted or even to have a selection dialog.

Reply:
Talked with @obkram in the morning. Going to make it an option on admin panel, also going to add everything else with checkboxes.